extensions/interactions/NumericInput/directives/numeric-input-validation.service.ts

```diff
@@ -190,8 +197,8 @@ export class NumericInputValidationService {
         stringValue = str + z;
       }
     }
-
-    if (stringValue.match(/\d/g).length > 15) {
+    const stringValueRegExp = stringValue.match(/\d/g);
+    if (stringValueRegExp === null || stringValueRegExp.length > 15) {
```

Comment:
I think we should return a different error depending if `stringValueRegExp` is null

Reply:
IstringValueRegExp returns `null` if the input contains anything other than a digit (0-9). I think the error message already asserts that scenario.